pkg/project/app_cloud/Taskfile.yml

```diff
@@ -203,9 +203,12 @@ tasks:
     aliases: [dispatcher:up]
     dir: dispatcher
     cmds:
-      # - sh aem/home/opt/sdk/dispatcher/bin/validate.sh dispatcher/src > aem/home/var/log/dispatcher-validate.log # uncomment if possible, see: https://github.com/adobe/aem-project-archetype/issues/1043
-      - rm -fr target/sdk && cp -r ../aem/home/opt/sdk/dispatcher target/sdk
+      - 'command -v docker || { echo "Docker is not installed! Visit: https://www.docker.com/products/docker-desktop"; exit 1; }'
+      - 'test -d ../aem/home/opt/sdk/dispatcher || { echo "Dispatcher SDK is not available. Try providing AEM SDK ZIP instead of JAR to directory: aem/home/lib"; exit 1; }'
+      - test -d target/sdk || rm -fr target/sdk && cp -r ../aem/home/opt/sdk/dispatcher target/sdk
+      # - sh target/sdk/bin/validate.sh src > ../aem/home/var/log/dispatcher-validate.log
```

Comment:
surprisingly I have no `validate.sh` unpacked / is is not included anymore in SDK zip?  I don't know.. Adobe missed that?